content/client-lib-development-guide/features.textile

```diff
@@ -866,7 +866,20 @@ h3(#channel-states-operations). @RealtimeChannel.state@ effects on channel opera
 
 h2(#push-notifications). Push notifications
 
-* @(RSH1)@ @Push#publish(recipient, data)@ performs a HTTP request to "/push/publish":/rest-api/#push-publish
+* @(RSH1)@ @Push#admin@ object provides the following interface:
+** @(RHS1a)@ @#publish(recipient, data)@ performs an HTTP request to "/push/publish":/rest-api#push-publish. Empty values for @recipient@ or @data@ should be immediately rejected. An end-to-end push notification test can be made using the special test-only @ablyChannel@ recipient. Additionally, tests should exist with valid and invalid recipient details
+** @(RHS1b)@ @#deviceRegistrations@ provides access to the admin @PushDeviceRegistrations@ object with the following methods:
+*** @(RHS1b1)@ @#get(deviceId)@ performs a request to "/push/deviceRegistrations/:deviceId":/rest-api#get-device-registration and returns a @DeviceDetails@ object if the @deviceId@ is found or results in a not found error if the device cannot be found
```

Comment:
BTW. So that you know, it already does this, see https://github.com/ably/realtime/blob/master/test/rest/push.js#L155-L168